src/components/organisms/Footer.scss

```diff
@@ -16,9 +16,9 @@
     color: $font-color;
   }
 
-  .container {
+  &__container {
 
-    .links-and-address {
+    &__links-and-address {
```

Comment:
Why BEM?

Reply:
This was how things were being done in frost-web, so I was just continuing that trend.